winter/pagination/sort/order_by.py

```diff
@@ -4,7 +4,11 @@
 from ...core.annotation_decorator import annotate_method
 
 
-def order_by(allowed_fields: typing.Iterable[str]):
+def order_by(allowed_fields: typing.Iterable[str], default: typing.Optional[str] = None):
     allowed_fields = frozenset(allowed_fields)
-    annotation = OrderByAnnotation(allowed_fields)
+    if default is not None:
+        assert all(field in allowed_fields for field in default.split(',')), (
```

Comment:
Looks like a third duplicate of the parsing logic, can we have a function `parse_sort(str) -> Sort` and use it everywhere?